Answers.md

```diff
@@ -1,2 +1,44 @@
 # Your responses to the short answer questions should be laid out here using Mark Down.
 ### For help with markdown syntax [Go here](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)
+
```

Comment:
These are great descriptions.  It definitely appears that you have a good understanding of the concepts.